srv/modules/runners/validate.py

```diff
@@ -217,7 +217,7 @@ def parse_list_from_string(list_str, delim=','):
         return [elem.strip() for elem in list_str.split(delim) if elem.strip()]
 
 
-LUMINOUS_VERSION = "11.2"
+NAUTILUS_VERSION = "14.0"
```

Comment:
I think I see `LUMINOUS_VERSION` still appearing in the file further down? That would become an undefined symbol?

Reply:
yep, fixing